### PR TITLE
Detailing our process for developing on VCR

### PR DESCRIPTION
**Pull request by krainboltgreene**
Feel free to add more.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  0. Run `bundle exec rake release`
+  0. Grab a drink
+
+Ancillary 1: We also have a thing called Relish, which I dislike immensley, but many people desire. It's on the last legs. You push the new relish documentation via `bundle exec rake relish`. I wish you the best of luck.
+
+Ancillary 2: At your leasure make a Release on Github with the changelog differences. It's very helpful for many people.
+
+## Changelog
+
+We use a very rudementry changelog system focusing on three primary things people care about: Breaking Changes, New Features, and Bug fixes. Anything outside of that is worthless for people looking at the changelog in my experience. Here's a snippet for a new changelog entry:
```
Comment (olleolleolle) [inline DEVELOPMENT.md:24]:
```suggestion
We use a very rudimentary changelog system focusing on three primary things people care about: Breaking Changes, New Features, and Bug fixes. Anything outside of that is worthless for people looking at the changelog in my experience. Here's a snippet for a new changelog entry:
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Development
+
+This file details as many imporant parts of the process that you may need to know about working on VCR, should any of us get hit by a bus.
```
Comment (olleolleolle) [inline DEVELOPMENT.md:3]:
```suggestion
This file details as many important parts of the process that you may need to know about working on VCR, should any of us get hit by a bus.
```

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  0. Denote changes since last release in changelog file (see changelog notation below)
+  0. Commit the changes with the body "Bumping to version X.Y.Z"
+  0. Push the branch
+  0. Make a pull request
+  0. Get it merged
+  0. Pull down the new master
+  0. Run `bundle exec rake release`
+  0. Grab a drink
+
+Ancillary 1: We also have a thing called Relish, which I dislike immensley, but many people desire. It's on the last legs. You push the new relish documentation via `bundle exec rake relish`. I wish you the best of luck.
```
Comment (olleolleolle) [inline DEVELOPMENT.md:18]:
```suggestion
Ancillary 1: We also have a thing called Relish, which I dislike immensely, but many people desire. It's on the last legs. You push the new relish documentation via `bundle exec rake relish`. I wish you the best of luck.
```

**Comment by krainboltgreene** [comment]
@olleolleolle I'll wait till after your release.

**Comment by olleolleolle** [comment]
In order to get forward on this, I took this path:

```
SKIP_RELISH=true bundle exec rake release
```

**Pull request merged**